pages/privacy-policy.tsx

```diff
@@ -0,0 +1,399 @@
+          understand this kuleana and work hard to protect your information and
+          to put you in control. It is Hawaiians in Technology's policy to
+          respect your privacy regarding any information we may collect from you
+          across our website,{" "}
+          <Link href="https://hawaiiansintech.org">hawaiiansintech.org</Link>,
+          and other sites we own and operate.
+        </p>
+      </section>
+      <section
+        className={`
+          mx-auto
+          max-w-7xl
+          px-4
+          text-xl
+          sm:my-12
+          sm:flex
+          sm:space-x-16
+          sm:px-8
+        `}
+      >
+        <aside
+          className={`
+            sm:sticky
+            sm:top-8
+            sm:shrink-0
+            sm:self-start
+          `}
+        >
+          <ul
+            className={`
+                  space-y-1
+                  py-4
+                  font-medium
+                `}
+          >
+            <li>
+              <Link
+                href="#information-we-collect"
+                className="text-xl font-bold"
+              >
+                Information we collect
+              </Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#joining-the-directory">Joining the Directory</Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#website-analytics">Website Analytics</Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#user-generated-content">User-Generated Content</Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#children-and-privacy">Children and Privacy</Link>
+            </li>
+            <li>
+              <Link
+                href="#your-rights-and-choices"
+                className="text-xl font-bold"
+              >
+                Your rights and choices
+              </Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#removal-from-the-directory">
+                Removal from the Directory
+              </Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#updates-and-notifications">
+                Updates and notifications
+              </Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#contact-us">Contact Us</Link>
+            </li>
+            <li>
+              <Link href="#security" className="text-xl font-bold">
+                Security
+              </Link>
+            </li>
+            <li className="pl-4">
+              <Link href="#data-retention">Data Retention</Link>
+            </li>
+            <li>
+              <p className="flex items-center gap-1 py-4 text-sm font-medium text-stone-800">
+                Last updated{" "}
+                <span className="inline-flex">
+                  <Pill>August 1, 2023</Pill>
```

Comment:
also, don't forget to change this date if it's not merged tomorrow

Reply:
Changed to tomorrow 🤞😅